hardhat.config.js

```diff
@@ -139,17 +138,31 @@ module.exports = {
     }
   },
 
-  solc: {
-    version: "0.6.6",
-    optimizer: require("./solcOptimiserSettings.js")
+  solidity: {
+    compilers: [{
+      version: "0.4.18",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }, {
+      version: "0.5.11",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }, {
+      version: "0.6.6",
+      settings: {
+        optimizer: require("./solcOptimiserSettings.js")
+      }
+    }]
   },
 
   paths: {
-    sources: "./contracts/sol6",
-    tests: "./test/sol6"
+    sources: "./contracts",
+    tests: "./test",
   },
 
   mocha: {
-    enableTimeouts: false
+    timeout: 0
```

Comment:
If 0 timeout, can remove the `mocha` field altogether?

Reply:
Removed

Reply:
Travis is getting timeout when `mocha` is removed, so adding it back